Common++/header/IpAddress.h

```diff
@@ -17,6 +17,17 @@
 #endif
 #endif
 
+#ifndef PCPP_DEPRECATED_MSG
```

Comment:
why do you create a new one. `IpAddress.h` should already have `PCPP_DEPRECATED`

Reply:
As far as I know `PCPP_DEPRECATED` does not allow to specify the message why certain feature is deprecated? `PCPP_DEPRECATED_MSG` allows to specify the message

Reply:
sure, I think it's better to have a message as well.